triveni-codev/GBWebApp-dev
# Justin_Outputs-PSIP-Tree-View-Hierarchy-Listing-In-View-Selected-Transport_1203576681646044

ASANA ticket: https://app.asana.com/0/1203486985240966/1203576681646042/f

**Description: created a function which will display the data of the selected transport in an output - PSIP type (Triveni Carousel)**
- amended treegrid UI view
- amended the  function call for manipulating the Transport data
- added some ej2-syncfusion-treegrid libraries
- added some icons and displayed it in the treegrid ~ dynamically


**TO TEST:**
- Create or Edit an Output **(Triveni Carousel type)** via angular or serverapp
- Associate a Transport in the Output
- Conduct a play-by-play edit in the Transport data (add Programs, Transport Stream, etc)
- Edit the newly added Output (with Transport) and click VIEW transport
- Check if the Treegrid view is correct and same with the serverapp

Note: CRUD of OUTPUT is an out-of-scope of this ticket, please disregard bugs you will encounter regarding this.

cc: @triveni-codev
